frontend/plan/reducers/friendships.js

```diff
@@ -0,0 +1,46 @@
+import {
+    UPDATE_FRIENDSHIPS_ON_FRONTEND,
+    SWITCH_ACTIVE_FRIEND,
+} from "../actions/friendshipUtil";
+
+const initialState = {
+    pulledFromBackend: false,
+    activeFriend: {},
```

Comment:
activeFriend refers to the friend's schedule the user is viewing? if so, consider renaming to something more clear